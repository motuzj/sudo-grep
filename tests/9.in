search word in multiple files + ignore case + invert match
-v --ignore-case "a" tests/texts/text1.txt tests/texts/text2.txt tests/texts/text3.txt tests/texts/text4.txt
