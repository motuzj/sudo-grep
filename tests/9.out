No sed do eiusmod tempor
Lectus ipsum dOLor purus non.
Lorem ipsum dolor
h'Yoggoth nufm'lutgh gof'
